server/services/GithubWorkflowService.js

```diff
@@ -126,13 +126,15 @@ const createGithubWorkflow = async ({ job, directory, gitSha }) => {
         jsonWebToken,
         GITHUB_APP_INSTALLATION_ID
     );
+    const browser = job.testPlanRun.testPlanReport.browser.name.toLowerCase();
```

Comment:
There are keys provided for the browsers but it's not currently passed from the database or available in the resolvers.

It's definitely not a problem here but wanted to raise that, in case it may lighten and simplify the load on this service trying to derive a key from `"VoiceOver for macOS".toLowerCase()` for example, in the future. It would also maintain predictable key maps for the browsers across this app and the workflow(s) being triggered.

Reply:
Oop! I realize now that I gave a bad and **very confusing** example because I must have viewed the `AT` table at the time I made the comment.

> There are keys provided for the browsers but it's not currently passed from the database or available in the resolvers.

I was wrong on this. Still in my confusion, I was referencing ATs **BUT** those keys also aren't being tracked in the db, instead being derived. This comment is then irrelevant to this PR.

> It seems like browser is just id/name (and the name to lower also seem pretty safe in this case)

It is safe!

But the point I'm making could still stand, in that there should be stored keys somehow. Not something for this PR but I think I'm just now becoming aware of a potential data consistency issue.

In the examples you gave here, when "Edge" gets added, they may want to present it as "Microsoft Edge" but the service (or other sections of the app) could want to resolve `"Microsoft Edge"` to `"edge"` for simplicity. Or if it ever comes to adding `"Safari (for iOS)"` -> `"safari_ios"`, it would mean renaming the current "Safari" browser being tracked to something more appropriate like "Safari (for macOS)" which could have been implicitly being thought of as `"safari_macos"` anyways.

All that to say, I'd feel safer using:
```javascript
job.testPlanRun.testPlanReport.browser.key;
```

VS

```javascript
job.testPlanRun.testPlanReport.browser.name.toLowerCase();
```

OR

```javascript
getBrowserKey(job.testPlanRun.testPlanReport.browser.name.toLowerCase());
```

If you or others agree to those keys, then that's a separate task to maybe make future needs easier.

Reply:
^ i don't feel entirely qualified to make this patch, the database stuff is still a bit of a mystery to me, but i do think that a "key" that isn't the "display name" here is probably the long-term "right" solution. (Followup issue?) I don't think we need to solve it to land this however.

Reply:
> (Followup issue?) I don't think we need to solve it to land this however.

Agreed! I can make a follow up task to discuss

Reply:
Just adding a +1 to what @howard-e is suggesting!

Reply:
#958